src/test/java/org/opentripplanner/updater/trip/TimetableSnapshotSourceTest.java

```diff
@@ -105,7 +104,7 @@ public void testGetSnapshot() throws InvalidProtocolBufferException {
       List.of(TripUpdate.parseFrom(cancellation)),
       feedId
     );
-    assertSame(snapshot, updater.getTimetableSnapshot());
+    assertNotSame(snapshot, updater.getTimetableSnapshot());
```

Comment:
Why did this change? Is it the new eager initialization logic?

Reply:
Yes exactly